index.html

```diff
@@ -926,13 +954,18 @@ <h2>Client-Managed Version Creation</h2>
          <h2>Replacing Contents from Mementos</h2>
           <blockquote class="informative">
             Non-normative note:
-            Using the ingest-by-reference mechanism, one can replace the contents of an <a>LDPRv</a> with that of
-            an <a>LDPRm</a> by providing it's URL as the <code>URL</code> parameter in a <code>Content-Type:
-            message/external-body</code> header.  For example, given an <a>LDPRm</a> with URL <code>
-            http://example.org/some/memento</code>, the full header would be <br/>
-            <code>Content-Type: message/external-body; access-type=URL; expiration=1;<br/>
-            &nbsp;&nbsp;&nbsp;&nbsp;URL="http://example.org/some/memento"</code>
+            Using the ingest-by-reference mechanism provided by the <code>external-content-handling=copy</code>
+            preference described in <a href="#external-content"></a>, servers may support replacement of the contents
+            of an <a>LDPRv</a> with that of an <a>LDPRm</a> by providing the <a>LDPRm</a>'s location as the target
```

Comment:
change "an" to "a"

Reply:
We currently use "an LDP..." 55 of 56 times in the spec.   I don't have a strong feeling about whether we should agree with the acronym or spelled-out form, but we should be consistent.

Reply:
I think I'm incorrect on this as well, it sounds like 'an' might the the accepted article.  So, ignore my comment.

Reply:
Thought it is expanding scope a wee bit, I'll change the one "a LDP-NR" that @escowles mentioned